### ISSUE TITLE
[Boss] Marax

### ISSUE THREAD

**Issue by warpdragon** [AngelArenaAllstars/suggestions#64]
# Marax

## Arena
A Stone platform.

## Boss
A great and mighty figure, like a body builder. Demon Wings, bare chest/arms or gauntlets (something bad-ass). Has a punching auto attack.

## Phase 1 (100%-66% HP)
Boss charges up, then punches outward. A giant **Blast Wave** shoots outward in a telegraphed direction, knocking enemies back and doing damage. Boss occasionally launches fireballs one-at-a-time into the air at a telegraphed point on the ground. They explode doing AoE damage when they land. These fireballs leave a medium AoE **Ring of Fire** on the ground after they explode. Players who cross through the ring take Medium damage and a DoT Debuff. 

## Phase 2 (66%-33% HP)
At this point, Boss can start casting **Warcry** giving him bonus armor and quickly causing a **Fury of Fists** to quickly pound the ground from the sky (like meteors), possibly shaking the screen. These Fists are dodgeable, but come at the players very fast and in large number, doing medium damage and slowing in their AoE when they hit.

## Phase 3 (33%-0% HP)
His **Warcry** becomes more powerful, giving him **Immolation** buff which does damage to any units in melee range and burning the ground behind him, leaving a **Trail of Fire** with the same effect and Debuff as **Ring of Fire**

**Comment by Haganeko** [comment]
References for **Blast Wave**, **Trail of Fire** and **Ring of Fire**?

**Comment by warpdragon** [comment]
Yeah I know. I should have used more "boring terminology", but I did actually describe each one in detail.

**Comment by Haganeko** [comment]
Come on. Keeping to the primer costs you nothing (but time and effort).
